Modules/SaltStackConfig/SaltStackConfig.psm1

```diff
@@ -0,0 +1,304 @@
+<#
+Copyright 2021 VMware, Inc.
+SPDX-License-Identifier: BSD-2-Clause
+#>
+Function Connect-SscServer {
+<#
+  .NOTES
+  ===========================================================================
+   Created by:	Brian Wuchner
+   Date:		November 27, 2021
+   Blog:		www.enterpriseadmins.org
+   Twitter:		@bwuch
+  ===========================================================================
+  .SYNOPSIS
+    This function will allow you to connect to a vRealize Automation SaltStack Config API.

```

Comment:
Is the official name of the API "vRealize Automation SaltStack Config API"? Is it only for configuring SaltStack?

Reply:
The product is called vRealize Automation SaltStack Config and this is for the RaaS API.  I've changed the comments from vRealize Automation SaltStack Config API to vRealize Automation SaltStack Config RaaS API to be more specific.